docs/4_troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+# 🛟 Troubleshooting
+
+This is a collection of errors and fixes for the ReVanced Manager application.
```

Comment:
The sentence style differs from the others. Here it is only referred to what the page contains. In other pages, you explicitly refer to the reader.